# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/src/jit/mips32/Architecture-mips32.h
# Commit: 1a73cfde72e8
# Full Hash: 1a73cfde72e8f4d19bdcfdbffa9fbc78bba01e24
# Author: Dmitry Bezhetskov <dbezhetskov@igalia.com>
# Date: 2021-09-28 09:54:33
# Regressor Bug: 1639153
# File Overlap Count: 1
# Description:
#   Bug 1639153 - Introduce indirect stubs to optimize call_indirect. r=lth
#   
#   This patch introduces indirect stubs.
#   An indirect stub is a stub that takes care of any switching activities needed for call_indirect.
#   Before this patch, we have to conservatively assume that any call_indirect's target can be from a foreign instance,
# ==============================================================================

diff -r da3c5d8492db -r 1a73cfde72e8 js/src/jit/mips32/Architecture-mips32.h
--- a/js/src/jit/mips32/Architecture-mips32.h	Tue Sep 28 05:16:23 2021 +0000
+++ b/js/src/jit/mips32/Architecture-mips32.h	Tue Sep 28 05:24:15 2021 +0000
@@ -21,6 +21,7 @@
 namespace jit {
 
 static const uint32_t ShadowStackSpace = 4 * sizeof(uintptr_t);
+static const uint32_t SizeOfReturnAddressAfterCall = 0;
 
 // These offsets are specific to nunboxing, and capture offsets into the
 // components of a js::Value.